### PR TITLE
[zh-cn] complete translation of web performance landing page

### PR DESCRIPTION
**Pull request by jasonren0403**
### Motivation

don't remain redundant outdated English content

### CODE REVIEW

File: files/zh-cn/learn/performance/index.md [hunk 1]
```diff
@@ -1,81 +1,60 @@
...
+  - : 现在你已经了解了一些性能指标，我们将深入了解性能工具、指标和 API，以及我们如何使性能成为 Web 开发工作流程的一部分。
+- [多媒体：图片](/zh-CN/docs/Learn/Performance/Multimedia)
+  - : 最容易提升网络性能的方式往往是媒体优化。根据每个用户代理的能力、尺寸和像素密度来提供不同的媒体文件是可能的。在这篇文章中，我们讨论了图像对性能的影响，以及减少每张图像发送字节数的方法。
+- [多媒体：视频](/zh-CN/docs/Learn/Performance/video)
+  - : 最容易提升网络性能的方式往往是媒体优化。在这篇文章中，我们讨论了视频内容对性能的影响，并涵盖了从背景视频中删除音轨等技巧，可以提高性能。
+- [JavaScript 性能最佳实践](/zh-CN/docs/Learn/Performance/JavaScript)
+  - : 如果使用得当，JavaScript 可以实现交互式和沉浸式的网络体验——或者它可以显著损害下载时间、渲染时间、应用内性能、电池寿命和用户体验。本文概述了一些应该考虑的 JavaScript 最佳实践，以确保即使是复杂的内容也能尽可能地执行。
+- [HTML 性能特性](/zh-CN/docs/Learn/Performance/HTML)
+  - : 一些属性和你的标记的源顺序会影响你的网站的性能。通过尽量减少 DOM 节点的数量，确保使用最佳的顺序和属性来包括样式、脚本、媒体和第三方脚本等内容，你可以大幅提高用户体验。这篇文章详细研究了如何使用 HTML 来确保最大的性能。
+- [CSS 性能特性](/en-US/docs/Learn/Performance/CSS)
```
Comment (JasonLamv-t) [inline files/zh-cn/learn/performance/index.md:44]:
```suggestion
- [CSS 性能特性](/zh-CN/docs/Learn/Performance/CSS)
```

File: files/zh-cn/learn/performance/index.md [hunk 2]
```diff
@@ -1,81 +1,60 @@
...
+  - : 在提高性能方面，CSS 可能是一个不太重要的优化重点，但有一些 CSS 特性对性能的影响比其他的更大。在这篇文章中，我们看看一些影响性能的 CSS 属性，以及处理样式的建议方法，以确保性能不受负面影响。
+- [字体与性能](/zh-CN/docs/Learn/Performance/Fonts)
+  - : 看看你是否需要包括外部字体，如果需要，如何包括你的设计所需的字体，对你的网站性能影响最小。
+- [移动端性能](/zh-CN/docs/Learn/Performance/Mobile)
+  - : 随着移动设备上的网络访问如此流行，所有的移动平台都有成熟的网络浏览器，但带宽、CPU 和电池寿命可能有限，考虑你的网络内容在这些平台上的性能是很重要的。本文探讨了针对移动设备的性能考虑。
+- [web 性能的商业案例](/zh-CN/docs/Learn/Performance/business_case_for_performance)
+  - : 开发人员可以做很多事情来提高性能，但多快才算够快？你怎样才能说服当权者相信这些努力的重要性？一旦优化了，你如何确保臃肿不会再出现？在这篇文章中，我们将探讨如何说服管理层，发展一种性能文化和性能预算，并介绍如何确保回归不会潜入你的代码库。
```
Comment (JasonLamv-t) [inline files/zh-cn/learn/performance/index.md:51]:
```suggestion
  - : 开发人员可以采取许多措施来提高系统的性能，但是什么样的速度才算快？如何说服管理层重视这些努力？一旦优化完成，如何确保系统不会再次出现性能问题？本文将探讨如何说服管理层、如何培养性能文化和制定性能预算，并介绍确保系统不会出现性能回退的方法。
```

File: files/zh-cn/learn/performance/index.md [hunk 3]
```diff
@@ -1,81 +1,60 @@
...
+- [多媒体：视频](/zh-CN/docs/Learn/Performance/video)
+  - : 最容易提升网络性能的方式往往是媒体优化。在这篇文章中，我们讨论了视频内容对性能的影响，并涵盖了从背景视频中删除音轨等技巧，可以提高性能。
+- [JavaScript 性能最佳实践](/zh-CN/docs/Learn/Performance/JavaScript)
+  - : 如果使用得当，JavaScript 可以实现交互式和沉浸式的网络体验——或者它可以显著损害下载时间、渲染时间、应用内性能、电池寿命和用户体验。本文概述了一些应该考虑的 JavaScript 最佳实践，以确保即使是复杂的内容也能尽可能地执行。
+- [HTML 性能特性](/zh-CN/docs/Learn/Performance/HTML)
+  - : 一些属性和你的标记的源顺序会影响你的网站的性能。通过尽量减少 DOM 节点的数量，确保使用最佳的顺序和属性来包括样式、脚本、媒体和第三方脚本等内容，你可以大幅提高用户体验。这篇文章详细研究了如何使用 HTML 来确保最大的性能。
+- [CSS 性能特性](/en-US/docs/Learn/Performance/CSS)
+  - : 在提高性能方面，CSS 可能是一个不太重要的优化重点，但有一些 CSS 特性对性能的影响比其他的更大。在这篇文章中，我们看看一些影响性能的 CSS 属性，以及处理样式的建议方法，以确保性能不受负面影响。
+- [字体与性能](/zh-CN/docs/Learn/Performance/Fonts)
+  - : 看看你是否需要包括外部字体，如果需要，如何包括你的设计所需的字体，对你的网站性能影响最小。
```
Comment (JasonLamv-t) [inline files/zh-cn/learn/performance/index.md:47]:
```suggestion
  - : 看看你是否需要引入外部字体，如果需要，如何以对网站性能影响最小的方式引入你的设计所需的字体。
```

File: files/zh-cn/learn/performance/index.md [hunk 4]
```diff
@@ -1,81 +1,60 @@
...
+- [移动端性能](/zh-CN/docs/Learn/Performance/Mobile)
+  - : 随着移动设备上的网络访问如此流行，所有的移动平台都有成熟的网络浏览器，但带宽、CPU 和电池寿命可能有限，考虑你的网络内容在这些平台上的性能是很重要的。本文探讨了针对移动设备的性能考虑。
```
Comment (JasonLamv-t) [inline files/zh-cn/learn/performance/index.md:49]:
```suggestion
  - : 随着移动设备上的网络访问普及，所有的移动平台都有成熟的网络浏览器，但可能存在带宽、CPU和电池寿命受限等问题，因此必须考虑在这些平台上提供的Web内容的性能。本文探讨了针对移动设备的性能考虑。
```

File: files/zh-cn/learn/performance/index.md [hunk 5]
```diff
@@ -1,81 +1,60 @@
...
+- [用户如何感知性能？](/zh-CN/docs/Learn/Performance/Perceived_performance)
+  - : 比你的网站有多快（以毫秒为单位）更重要的是你的用户认为你的网站有多快。这些感知受到实际页面加载时间、空闲、对用户互动的响应能力、滚动和其他动画的流畅度的影响。在这篇文章中，我们将讨论各种加载指标、动画和响应性指标，以及改善用户感知的最佳做法，如果不是实际时间的话。
+- [性能的衡量](/zh-CN/docs/Learn/Performance/Measuring_performance)
+  - : 现在你已经了解了一些性能指标，我们将深入了解性能工具、指标和 API，以及我们如何使性能成为 Web 开发工作流程的一部分。
+- [多媒体：图片](/zh-CN/docs/Learn/Performance/Multimedia)
+  - : 最容易提升网络性能的方式往往是媒体优化。根据每个用户代理的能力、尺寸和像素密度来提供不同的媒体文件是可能的。在这篇文章中，我们讨论了图像对性能的影响，以及减少每张图像发送字节数的方法。
+- [多媒体：视频](/zh-CN/docs/Learn/Performance/video)
+  - : 最容易提升网络性能的方式往往是媒体优化。在这篇文章中，我们讨论了视频内容对性能的影响，并涵盖了从背景视频中删除音轨等技巧，可以提高性能。
+- [JavaScript 性能最佳实践](/zh-CN/docs/Learn/Performance/JavaScript)
+  - : 如果使用得当，JavaScript 可以实现交互式和沉浸式的网络体验——或者它可以显著损害下载时间、渲染时间、应用内性能、电池寿命和用户体验。本文概述了一些应该考虑的 JavaScript 最佳实践，以确保即使是复杂的内容也能尽可能地执行。
```
Comment (JasonLamv-t) [inline files/zh-cn/learn/performance/index.md:41]:
```suggestion
  - : 如果使用得当，JavaScript 可以实现交互式和沉浸式的网络体验——或者它可以显著增加下载时间、渲染时间并损害应用性能、电池寿命和用户体验。本文概述了一些应该考虑的 JavaScript 最佳实践，以确保即使是复杂的内容也能尽可能地执行。
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
一些建议

File: files/zh-cn/learn/performance/index.md (hunk 4 above)
Comment (jasonren0403) [reply files/zh-cn/learn/performance/index.md:49]:
```suggestion
  - : 随着移动设备上的网络访问普及，所有的移动平台都有成熟的网络浏览器，但可能存在带宽、CPU 和电池寿命受限等问题，因此必须考虑在这些平台上提供的 Web 内容的性能。本文探讨了针对移动设备的性能考虑。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**